src/common/wrap/wrap-events.js

```diff
@@ -79,7 +76,32 @@ export function wrapEvents(sharedEE) {
 
   return ee
 }
+/**
+ * Find the base prototype of 'object' that has its own "addEventListener" property, and run some function on it.
+ * @param {Object} object - the initial object to traverse prototype chain on
+ * @param {Function} cb - the function to run on the ancestral object once found, accepts an object as a arg
+ * @param {Array} rest - [optional] any additional arguments to pass to the cb
+ */
+function findEventListenerProtoAndCb(object, cb, ...rest) {
+  let step = object;
+  // eslint-disable-next-line
+  while (step && !step.hasOwnProperty(ADD_EVENT_LISTENER)) { step = Object.getPrototypeOf(step) }
```

Comment:
`getPrototypeOf` can throw a `TypeError` exception. 

```suggestion
  while (typeof step === 'object' && !Object.prototype.hasOwnProperty.call(step, ADD_EVENT_LISTENER)) {
    step = Object.getPrototypeOf(step)
  }
```

Reply:
We just want to make sure `step` is an actual object right? Ok.

Reply:
Yup....and also eslint will start complaining about calling `hasOwnProperty` from the target object instead of using `Object.prototype.hasOwnProperty.call(....)`